apps/cli/package.json

```diff
@@ -5,23 +5,32 @@
   "main": "index.js",
   "type": "commonjs",
   "scripts": {
-    "build": "tsc",
-    "watch": "tsc -w",
-    "start": "node dist/index.js"
+    "build": "swc src --out-dir dist",
+    "start": "node dist/src/index.js",
```

Comment:
We will need a watch script in here. Maybe try including nodemon if swc doesnt have a watch feature. We won't need the dev script in here.

Reply:
why do we need to watch script it's a cli we need to type the command every time to run it, no need for watch script